### PR TITLE
Chelakhov leonid/hw6

### PR DESCRIPTION
**Pull request by Chelakhovl**

### CODE REVIEW

File: Chelakhovl/HW6/task2.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+login = ""
+while True :
```
Comment (kolyasalubov) [inline Chelakhovl/HW6/task2.py:2]: good job

File: Chelakhovl/HW6/TASK1.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+l1 = []
```
Comment (kolyasalubov) [inline Chelakhovl/HW6/TASK1.py:1]: excellent work

**Review by kolyasalubov — APPROVED** [review]

File: Chelakhovl/HW6/TASK1.py (hunk 2 above)
Comment (Chelakhovl) [reply Chelakhovl/HW6/TASK1.py:1]: Thank YOU

File: Chelakhovl/HW6/task2.py (hunk 1 above)
Comment (Chelakhovl) [reply Chelakhovl/HW6/task2.py:2]: THANKS =)

**Pull request merged**